src/contexts/jsvOptions.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+type JSVOptions = {
+  defaultExpandedDepth: number;
+  maxRows: number;
```

Comment:
Thinking about this.
Does it make any sense to keep this?
This was mostly useful for instrumenting tree-list to render X amount of rows at the same time.
Now, you should be able to control it with plain styles.

Reply:
Absolutely not, it doesn't even work. I forgot to remove it, will do. :D